martini.go

```diff
@@ -57,6 +57,12 @@ func (m *Martini) Action(handler Handler) {
 	m.action = handler
 }
 
+// Action sets the logger
```

Comment:
Should be `Logger sets the logger`. Anyway, I'd name the method `SetLogger`. By the name, `Logger` suggests that it's a getter.